src/common/models/data-cube/data-cube.ts

```diff
@@ -704,6 +704,10 @@ export class DataCube implements Instance<DataCubeValue, DataCubeJS> {
     // No need for the overrides
     value.attributeOverrides = null;
 
+    if (value.options.druidContext) {
```

Comment:
I thought that we will leave options unchanged. There's no critical data in `druidContext`.

Reply:
It seems unused. I've put some data in context, but it is not displayed in "Display Druid query" modal. And no errors occur if it is not sent in `External` object, where the other two (customTransforms and customAggregations) are passed.